src/NuGet.Clients/VsExtension/project.json

```diff
@@ -13,8 +14,8 @@
     "Microsoft.VisualStudio.Shell.Interop.8.0": "8.0.50727",
     "Microsoft.VisualStudio.Shell.Interop.9.0": "9.0.30729",
     "Microsoft.VisualStudio.Threading": "14.0.50702",
-    "Microsoft.VisualStudio.OLE.Interop": "7.10.6070",
-    "Microsoft.VSSDK.BuildTools": "14.0.23107"
+    "Microsoft.VSSDK.BuildTools": "14.0.23107",
```

Comment:
Do these pull in extra 3rd party dependencies?
Are any of these not already vetted by LCA for OSS?